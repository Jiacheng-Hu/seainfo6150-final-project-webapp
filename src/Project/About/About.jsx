import React from "react";
import banner from "../images/banner3.jpg";
const About = () => {
    return (
        <section className="container">
            <img src={banner} alt="hero" className=" img-responsive"/>
            <div className="about-content text-left">
                <h1 className="about-title">Experience is at the core of all we do</h1>
                <p className="about-text">
                    At Northeastern, experience is our essence and ethos. It’s what you gain when you make the world your classroom, your laboratory, and your platform to create change or grow your enterprise. Throughout our university network, experience draws you into society and compels you to solve its complex challenges. It makes you agile and able to reinvent yourself. To find ways of doing things differently, and better. And to seize opportunities as they unfold—anytime, anywhere.
                </p>
            </div>
        </section>
    )
}

export default About;