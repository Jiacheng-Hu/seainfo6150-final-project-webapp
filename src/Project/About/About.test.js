import React from "react";
import About from "./About";

describe("About tests", () => {
    it("renders correctly", () => {
        const {container} = render(<About/>);
        expect(container).toMatchSnapshot();
    });
});