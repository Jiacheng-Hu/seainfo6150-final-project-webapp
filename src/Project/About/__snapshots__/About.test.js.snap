// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About tests renders correctly 1`] = `
<div>
  <section
    class="container"
  >
    <img
      alt="hero"
      class=" img-responsive"
      src="banner3.jpg"
    />
    <div
      class="about-content text-left"
    >
      <h1
        class="about-title"
      >
        Experience is at the core of all we do
      </h1>
      <p
        class="about-text"
      >
        At Northeastern, experience is our essence and ethos. It’s what you gain when you make the world your classroom, your laboratory, and your platform to create change or grow your enterprise. Throughout our university network, experience draws you into society and compels you to solve its complex challenges. It makes you agile and able to reinvent yourself. To find ways of doing things differently, and better. And to seize opportunities as they unfold—anytime, anywhere.
      </p>
    </div>
  </section>
</div>
`;
